packages/mattermost-redux/src/store/configureStore.ts

```diff
@@ -1,33 +1,37 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {createStore, applyMiddleware, Store} from 'redux';
+import {createStore, applyMiddleware, Store, combineReducers} from 'redux';
+import thunk from 'redux-thunk';
+import {composeWithDevTools} from 'redux-devtools-extension/developmentOnly';
```

Comment:
`developmentOnly` embeds the necessary `process.env.NODE_ENV` checks so that tree-shaking both disable and removes the unnecessary code.

At some point in the future, I'd love to consider something like `logOnlyInProduction` to give acecss to the tools without any performance overhead. But this PR is meant as a no-op :)